DGRunkeeperSwitch/DGRunkeeperSwitch.swift

```diff
@@ -64,7 +64,18 @@ open class DGRunkeeperSwitch: UIControl {
         set { selectedBackgroundView.backgroundColor = newValue }
         get { return selectedBackgroundView.backgroundColor }
     }
-    
+	
+	@IBInspectable
+	// Set a custom corner radius or use the default
+	open var customCornerRadius: CGFloat? {
```

Comment:
Yeah, I have renamed it to `cornerRadius`. 
I am not totally sure what you mean with the `rounded` thing. If you don't set the `cornerRadius`, it will be set to the default in `finishInit()`. Then the user can specify 0, 8 or anything else to set the desired `cornerRadius`. I guess that is fine ?